packages/beacon-node/test/unit/db/api/repositories/dataColumn.test.ts

```diff
@@ -50,14 +50,15 @@ describe("block archive repository", () => {
     const slot = dataColumn.signedBlockHeader.message.slot;
     const blob = ssz.deneb.Blob.defaultValue();
     const commitment = ssz.deneb.KZGCommitment.defaultValue();
+    const kzgProof = ssz.deneb.KZGProof.defaultValue();
     const singedBlock = ssz.fulu.SignedBeaconBlock.defaultValue();
 
     singedBlock.message.body.blobKzgCommitments.push(commitment);
     singedBlock.message.body.blobKzgCommitments.push(commitment);
     singedBlock.message.body.blobKzgCommitments.push(commitment);
     const allDataColumnSidecars = computeDataColumnSidecars(config, singedBlock, {
       blobs: [blob, blob, blob],
-      kzgProofs: [commitment, commitment, commitment],
+      kzgProofs: Array.from({length: 3 * NUMBER_OF_COLUMNS}, () => kzgProof),
```

Comment:
nice catch but sizes might be same i guess before for typescript to not flag it

Reply:
Now that `computeDataColumnSidecars` reuses the proofs from the contents instead of computing them the input proofs need to contain one per cell. This test fails if there is only one proof per blob